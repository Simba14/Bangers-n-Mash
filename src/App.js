import React, { Component } from 'react';

class App extends Component {
  render() {
    return (
      <div className="App">
        <div className="App-header">
          <img src="BR_Logo.jpg" className="App-logo" alt="logo" />
        </div>
      </div>
    );
  }
}

export default App;
