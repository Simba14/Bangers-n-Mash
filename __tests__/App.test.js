import React from 'react';
import ReactDOM from 'react-dom';
import App from '../src/App';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
});

it('renders App correctly', () => {
  const tree = renderer.create(
    <App />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
