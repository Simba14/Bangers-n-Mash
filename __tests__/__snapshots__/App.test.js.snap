// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App correctly 1`] = `
<div
  className="app"
>
  <div
    className="app-header"
  >
    <h1>
      Boiler Room Picks
    </h1>
  </div>
  <div
    className="app-body"
  >
    <div
      className="video-display"
    >
      <div
        className="col-md-6 loading-image"
        id={undefined}
      >
        <div
          style={
            Object {
              "animation": "anim_11497540491071 1s 0.1s infinite cubic-bezier(.2,.68,.18,1.08)",
              "animationFillMode": "both",
              "backgroundColor": "#26A65B",
              "borderRadius": "2px",
              "display": "inline-block",
              "height": "35px",
              "margin": "2px",
              "verticalAlign": undefined,
              "width": "4px",
            }
          }
        />
        <div
          style={
            Object {
              "animation": "anim_11497540491071 1s 0.2s infinite cubic-bezier(.2,.68,.18,1.08)",
              "animationFillMode": "both",
              "backgroundColor": "#26A65B",
              "borderRadius": "2px",
              "display": "inline-block",
              "height": "35px",
              "margin": "2px",
              "verticalAlign": undefined,
              "width": "4px",
            }
          }
        />
        <div
          style={
            Object {
              "animation": "anim_11497540491071 1s 0.30000000000000004s infinite cubic-bezier(.2,.68,.18,1.08)",
              "animationFillMode": "both",
              "backgroundColor": "#26A65B",
              "borderRadius": "2px",
              "display": "inline-block",
              "height": "35px",
              "margin": "2px",
              "verticalAlign": undefined,
              "width": "4px",
            }
          }
        />
        <div
          style={
            Object {
              "animation": "anim_11497540491071 1s 0.4s infinite cubic-bezier(.2,.68,.18,1.08)",
              "animationFillMode": "both",
              "backgroundColor": "#26A65B",
              "borderRadius": "2px",
              "display": "inline-block",
              "height": "35px",
              "margin": "2px",
              "verticalAlign": undefined,
              "width": "4px",
            }
          }
        />
        <div
          style={
            Object {
              "animation": "anim_11497540491071 1s 0.5s infinite cubic-bezier(.2,.68,.18,1.08)",
              "animationFillMode": "both",
              "backgroundColor": "#26A65B",
              "borderRadius": "2px",
              "display": "inline-block",
              "height": "35px",
              "margin": "2px",
              "verticalAlign": undefined,
              "width": "4px",
            }
          }
        />
      </div>
    </div>
    <div
      className="video-list"
    >
      <ul
        className="col-md-3 video-list"
      >
        <h4
          className="list-heading"
        >
          I Was In The Crowd
        </h4>
      </ul>
    </div>
    <div
      className="video-list"
    >
      <ul
        className="col-md-3 video-list"
      >
        <h4
          className="list-heading"
        >
          All Time Favourites
        </h4>
      </ul>
    </div>
  </div>
</div>
`;
